src/util/usePanelParams.tsx

```diff
@@ -46,11 +66,20 @@ const usePanelParams = (): PanelHelper => {
     instance: params.get("instance"),
     profile: params.get("profile"),
     project: params.get("project") ?? "default",
+    identity: params.get("identity"),
+    group: params.get("group"),
+    idpGroup: params.get("idp-group"),
 
     clear: () => {
       clearParams();
     },
 
+    clearPanel: () => {
+      const newParams = new URLSearchParams(params);
+      newParams.delete("panel");
+      setParams(newParams);
+    },
```

Comment:
This is missing to reset the additional params that were added for the panel. I.e. open the panel to manage groups for an identity, and close the panel. This will leave behind a `identity=foo_10%40foo.com` parameter in the url that should be cleared as well.